### ISSUE TITLE
Doors sometimes fly through the map!

### ISSUE THREAD

**Issue by Xylemon** [fortressforever/fortressforever#151]
I've noticed that doors can sometimes open and fly through the map. They return to their spot, but it's still very strange and interferes with the gameplay.

**Comment by squeek502** [comment]
Yeah, this is an ancient Source engine bug. If there was any sort of reliable way to reproduce it, it might be able to be fixed, but I'm not sure there is.

**Comment by WiFiDi** [comment]
me and blacknight were messing with doors on a map by putting buildibles inbetween the doors would cause them to bug out, i don't think thats the same bug but id figure id mention it.just in case.

damn i forgot ilost all my screens when i delted FF 2.46 :(
